prv-net-helper.in

```diff
@@ -18,6 +18,9 @@ function up {
 	offset=$4
 
 	echo "Adding VLANs $2 - $3"
+	if [ ! -f /proc/net/vlan/config ]; then
+		modprobe 8021q
```

Comment:
`modprobe` should probably be considered an external executable and be autodetected in ./configure and be overwritable be the `MODPROBE` variable like the rest of the external executables.